lib/dentaku/syntax/skeleton.rb

```diff
@@ -3,10 +3,15 @@ module Syntax
     module Skeleton
       class Base
         def root?(*)   false; end
+
         def nested?(*) false; end
+
+        # TODO - clause should be deprecated, but is still in use elsewhere.
```

Comment:
Turns out `#clause?` is actually needed - it just probably shouldn't be in the block with the rest of these. Used here in `parse_case`:

https://github.com/opencounter/dentaku/blob/master/lib/dentaku/syntax/parser.rb#L274